lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -153,14 +153,22 @@ class TurboGraft.Remote
       initiator: @initiator
       xhr: xhr
 
-    if @refreshOnError
-      Page.refresh
-        response: xhr
-        onlyKeys: @refreshOnError
-    else if @refreshOnErrorExcept
-      Page.refresh
-        response: xhr
-        exceptKeys: @refreshOnErrorExcept
-    else
+    if TurboGraft.hasTGAttribute(@initiator, 'tg-remote-norefresh')
```

Comment:
interesting handling here, do you want to change success path to also trigger an event?